kitchen-sink-example.yaml

```diff
@@ -112,6 +112,26 @@ propagators: [tracecontext, baggage, b3, b3multi, jaeger, xray, ottrace]
 
 # Configure tracer provider.
 tracer_provider:
+  sampler:
+    # Choose sampler id. Must be entry point name of sampler
+    sampler_id: parent_based
```

Comment:
I decided sampler_id is clearer than "type" which we used in our weekly sig @jack-berg Thoughts?